TESTING.asciidoc

```diff
@@ -313,39 +321,28 @@ a bit more performance out of the process:
 +
-. Validate your installed dependencies:
+
+. You can run all of the OS packaging tests with `./gradlew distroTest`. This
+will cause Gradle to build the tar, zip, and deb packages and all the plugins.
+It will then run the tests on every available system. This will take a very 
+long time.
 +
--------------------------------------
-./gradlew :qa:vagrant:vagrantCheckVersion
--------------------------------------
+Fortunately, the various systems under test have their own Gradle tasks under
+`qa/os`. To find out what packaging combinations can be tested on a system, run
+the `tasks` task.  For example:
 +
-. Download and smoke test the VMs with `./gradlew vagrantSmokeTest` or
-`./gradlew -Pvagrant.boxes=all vagrantSmokeTest`. The first time you run this it will
-download the base images and provision the boxes and immediately quit. Downloading all
-the images may take a long time. After the images are already on your machine, they won't
-be downloaded again unless they have been updated to a new version.
+----------------------------------
+./gradlew :qa:os:ubuntu-1804:tasks
+----------------------------------
 +
-. Run the tests with `./gradlew packagingTest`. This will cause Gradle to build
-the tar, zip, and deb packages and all the plugins. It will then run the tests
-on ubuntu-1604 and centos-7. We chose those two distributions as the default
-because they cover deb and rpm packaging and SyvVinit and systemd.
-
-You can choose which boxes to test by setting the `-Pvagrant.boxes` project property. All of
-the valid options for this property are:
-
-* `sample` - The default, only chooses ubuntu-1604 and centos-7
-* List of box names, comma separated (e.g. `oel-7,fedora-28`) - Chooses exactly the boxes listed.
-* `linux-all` - All linux boxes.
-* `windows-all` - All Windows boxes. If there are any Windows boxes which do not
-have images available when this value is provided, the build will fail.
-* `all` - All boxes we test. If there are any boxes (e.g. Windows) which do not have images
-available when this value is provided, the build will fail.
-
-For a complete list of boxes on which tests can be run, run `./gradlew :qa:vagrant:listAllBoxes`.
-For a list of boxes that have images available from your configuration, run
-`./gradlew :qa:vagrant:listAvailableBoxes`
-
-Note that if you interrupt gradle in the middle of running these tasks, any boxes started
+If you want a quick test of the tarball and RPM packagings for Centos 7, you
+would run:
++
+-------------------------------------------------------------------------------------------------
+./gradlew :qa:os:centos-7:distroTest.default-rpm :qa:os:centos-7:distroTest.default-linux-archive
+-------------------------------------------------------------------------------------------------
+
+Note that if you interrupt Gradle in the middle of running these tasks, any boxes started
 will remain running and you'll have to stop them manually with `./gradlew stop` or
```

Comment:
This should be `./gradlew --stop`. @rjernst I assume it's the reaper process that goes back and kills these VMs when the daemon is killed?

Reply:
Yes the reaper will shut the VMs down.